### PR TITLE
Add TDP 44 & Update Headers

### PR DESCRIPTION
**Pull request by mzero2nine**
Hi @dimsome,

can you please review below and merge if possible? Thank you so much!

All the best,
m09

### CODE REVIEW

File: TDP/tdp-44.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+author: mZeroNine (@mzero2nine)
+discussions-to: https://forum.mstable.org
+created: 2022-06-20
+---
+
+## Simple Summary
+
+It is proposed to restructure the mStableDAO in order to increase its efficiency as well as overall ability to scale in the future without further re-adjustments down the line in order to prepare for the release of mStable v2. 
+
+Since the restructuring process as a whole is extensive, it is also proposed to split the mStableDAO restructuring into multiple TDPs in order to maintain explicitness in the voting outcome as well as ensuring a digestible format for Meta Governors. The entire process will contain 4 TDPs in total and is planned to conclude by the 3rd week of August 2022 the latest.
```
Comment (dimsome) [inline TDP/tdp-44.md:14]: Date in the proposal is not relevant to the proposal. I would not specify this.

File: TDP/tdp-44.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Abstract
+
+This first TDP is targeted at the overall proposition to begin the restructuring of the mStableDAO. 
+
+The key actions proposed include:
+
+- Introducing a general universal contributor title which would involve removing current individual rules and holding a full election process (part of the next TDP) for each of the available seats inside of the respective DAO. Both DAOs will operate under a 4/6 multisig.
+- Merging the Asset Management subDAO with the TreasuryDAO and absorb primary funding functions of the Funding subDAO back into the TreasuryDAO.
+- Creation of a new “Product/Builder subDAO” to separate the ProtocolDAO from the full-time contributor's group that is responsible for the continuous roll-out and operations of the mStable product line. The creation of this subDAO will ensure the long-term success of the protocol while preventing any centralized points of failure or sensitivity to external threats
```
Comment (dimsome) [inline TDP/tdp-44.md:29]: `Product/Builder subDAO` Is that the final name?

File: TDP/tdp-44.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- A disconnect between the way the core team works together and the structure of the DAO
+- A lack of clear purpose and deliverables for entities funded by the mStableDAO
+
+This proposal aims to address the inefficiencies outlined above through simplification of DAO structure and clarification of the purpose, responsibilities, and decision-making processes within each entity.
+
+Aside from aligning with a new vision and focus for mStable, the timing of this review also coincides with a number of key dates that would have already required individual governance processes, so it’s the perfect time to consolidate all of these requests in this restructuring proposal.
+
+## Specification
+
+1.) Begin parallel rollout of TDP 44 for the election process, TDP 44 for the GrantsDAO and Community subDAO consolidation & review of the GrantsDAOs success after it’s first year of operation, as well as TDP 44 for the proposed compensation model under which the mStableDAO will fund its workforce.
```
Comment (dimsome) [inline TDP/tdp-44.md:55]: TDP 45 for the election process, TDP 46 GrantsDAO and Community subDAO consolidation & review of the GrantsDAOs success after it’s first year of operation, as well as TDP 47 for the proposed compensation model under which the mStableDAO will fund its workforce.

Comment (dimsome) [reply TDP/tdp-44.md:55]: add links later to the proposals once they are up :)

File: TDP/tdp-44.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- Whether or not Meta Governors wish the mStableDAO to be restructured as described below
+- The format of having subsequent TDPs that will resolve more granular details of the restructuring process as a whole
+
+## Abstract
+
+This first TDP is targeted at the overall proposition to begin the restructuring of the mStableDAO. 
+
+The key actions proposed include:
+
+- Introducing a general universal contributor title which would involve removing current individual rules and holding a full election process (part of the next TDP) for each of the available seats inside of the respective DAO. Both DAOs will operate under a 4/6 multisig.
```
Comment (dimsome) [inline TDP/tdp-44.md:27]: Is this something that would be specified in the next TDP?

**Review by dimsome — CHANGES_REQUESTED** [review]
So in a nutshell, this TDP is just asking whether we want to restructure or not? Why not propose the how directly in the proposal?

File: TDP/tdp-44.md (hunk 3 above)
Comment (mzero2nine) [reply TDP/tdp-44.md:55]: I thought we agreed to name them all TDP 44?

File: TDP/tdp-44.md (hunk 1 above)
Comment (mzero2nine) [reply TDP/tdp-44.md:14]: > So in a nutshell, this TDP is just asking whether we want to restructure or not? Why not propose the how directly in the proposal?

I'm not sure I follow? 

> Date in the proposal is not relevant to the proposal. I would not specify this.
Roger, can change once above comment is sorted :)

File: TDP/tdp-44.md (hunk 2 above)
Comment (dimsome) [reply TDP/tdp-44.md:29]: I am fine with this, but since it is the proposal, it should imo specify the name :D

**Review by dimsome — APPROVED** [review]
All good now.
just might want to fix a name for the product/builder DAO so it has an actual name :)

**Pull request merged**